lib/queries/checkEditable.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = function (userId, chalId) {
+  return [
+    'SELECT',
+    'challenges.id as chid, challenges.title as chal_title,',
```

Comment:
I don't think we need the title or the first_name on this query, we could only return the id